lobsterpy/cli.py

```diff
@@ -235,91 +258,156 @@ def get_parser() -> argparse.ArgumentParser:
         dest="save_plot",
         help="Save plot to file",
     )
+    plotting_group.add_argument("--title", type=str, default="", help="Plot title")
     plotting_group.add_argument(
-        "--width", type=float, default=None, help="Plot width in inches"
+        "-wd",
```

Comment:
Same comment as before. I guess standards should exist for the short notation

Reply:
I checked into this, usually -w is used. The short notation for `height `arg will be `-h` which creates a conflict with default `-h` arg for helo. So I feel `-wd `and `-ht` will keep it bit consistent.